tests/local_worker_test.py

```diff
@@ -24,20 +24,20 @@
 
 class LocalWorkerTest(unittest.TestCase):
 
-    @timeout(TEST_TIMEOUT, use_signals=False)
+    @timeout(TEST_TIMEOUT, use_signals=True)
     def test_runner_local(self):
```

Comment:
Had to change this because otherwise Python produces some pretty weird error on the VM level.